src/envoy/http/authn/origin_authenticator.cc

```diff
@@ -31,68 +31,56 @@ OriginAuthenticator::OriginAuthenticator(FilterContext* filter_context,
     : AuthenticatorBase(filter_context), policy_(policy) {}
 
 bool OriginAuthenticator::run(Payload* payload) {
-  bool success = false;
+  bool triggered_success = false;
+  bool triggered = false;
 
   if (policy_.origins_size() == 0) {
```

Comment:
This can be one if.

Reply:
I want to avoid getting the path when origins size is 0.

Reply:
It doesn't make any different, does it? If the list is empty, the loop won't do anything thing, hence two booleans won't be changed.

Reply:
Right. I mean I want to avoid the line 51-58 that gets the path from the header. That part is not needed when the loop won't do anything.

Reply:
Removed the else part.